### ISSUE TITLE
Editing a bookmark in long list causes the browser to jump to top of page

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#16811]
### Description: 
Reported behavior is that when bookmarks are edited -- deleted, modified, etc. -- the act of editing causes the browser to jump to the top of the page, rather than retain your current position on the page. 

### Steps to Reproduce 
  1. Ensure you have enough bookmarks to necessitate scrolling down.
  2. Edit a bookmark further down the list
  3.

**Actual result:** 
Browser jumps to top of bookmarks page after edit.

**Expected result:**
Browser should stay in current position in the list after editing.

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Easily.

**Brave Version:**
v1.27.84, v1.26.74

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? No

## Miscellaneous Information:

User Report: https://community.brave.com/t/when-editing-bookmarks-the-list-scrolls-back-to-the-top-automatically/263322/2

**Comment by bsclifton** [comment]
Not clear if this is in brave://bookmarks/ or in bookmarks toolbar; followed up to user w/ question

**Comment by bsclifton** [comment]
update: this is on brave://bookmarks/ 👍

**Comment by Brave-Matt** [comment]
Added the macOS label here as I'm seeing reports of this behavior on macOS as well.

**Comment by RexBeans** [comment]
It is in brave://bookmarks/
for instance, if I were to right-click a bookmark from the bookmark toolbar, and choose "Bookmark Manager" from the drop-down.
It's not just when editing or moving the bookmarks, even during navigation of the list itself it jumps back to the top.

**Comment by tortino** [comment]
yes, it's probably after sync completes that the page is refreshed, and goes back to the top

**Comment by PorthosSam** [comment]
Still happening in Brave Version 1.33.106 under Windoze 10.

**Comment by m77e4t** [comment]
@424344 
I think you may be right. I tried to replicate it on another brave browser not connected to sync chain (it was brave beta) and I could reproduce it, 
But, on a brave browser connected to sync chain, a normal edit or deletion of a particular bookmark jumps the screen to the top.

**Comment by acepilot1944** [comment]
This issue is still happening on Brave version 1.36.117 & macOS Big Sur. Sometimes Brave automatically scrolls to the top of the page when a Bookmark is edited or deleted, and sometimes it does this WITHOUT such interaction, during normal scrolling.

**Comment by PorthosSam** [comment]
Thanks, you guys.  It's nice to feel like I'm not alone! 
Sam Johnson
 

    On Saturday, March 19, 2022, 10:19:36 PM MDT, acepilot1944 ***@***.***> wrote:  
 
 


This issue is still happening on Brave version 1.36.117 & macOS Big Sur. Sometimes Brave automatically scrolls to the top of the page when a Bookmark is edited or deleted, and sometimes it does this WITHOUT such interaction, during normal scrolling.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by rebron** [comment]
There's a workaround for this issue for now by using the bookmarks panel in sidebar. There won't be any jumpiness via using that list to manage and view bookmarks.

**Comment by 424344** [comment]
> using the bookmarks panel in sidebar.

yes, but for people with many nested folders and bookmarks the panel is not big enough: I just see the beginning of bookmarks names, so it's almost useless for me right now. 

you should make the panel resizable.

**Comment by davidgenesiscruz** [comment]
> There's a workaround for this issue for now by using the bookmarks panel in sidebar. There won't be any jumpiness via using that list to manage and view bookmarks.

This is nice. But the sidebar doesn't appear when in fullscreen mode so I cannot use this workaround, sadly.

**Comment by zpoison** [comment]
Still no solution? Same problem 6 month later, keeps jumping on top of page

**Comment by acepilot1944** [comment]
Forgive me if this is hijacking the thread, but until this gets solved (or even after), it may be useful to use a third-party extension as an alternative bookmark manager. I use Raindrop. This introduces some privacy concerns, rather antithetical to the main point of Brave, but it is very convenient to be able to sort bookmarks in more ways than Chromium-based browsers allow. For example, sorting new-to-old puts your recent bookmarks on top instead of at the bottom of a potentially massive list. Sorting by type (videos, podcasts, articles, etc.) is also useful.

**Comment by RexBeans** [comment]
I welcome the input, because as you can see from Brave Community's 20 exact same threads about this problem, solving this is not a priority. They can't even get their BAT payment system to work after nearly a year of randomly not paying people what they were promised. My solution to that was to turn off Brave ads, because there was no longer any incentive to use them if there's a 50/50 chance that Brave will either "forget" or never deliver my BAT unless I make repeated attempts to personally beg for what should be an automated payout. Brave would absolutely dominate if they could get their ____ together.

**Comment by Baneybo** [comment]
A year and a half later and the problem persists.
The most user-unfriendly aspect of using Brave; makes me want to jump ship every time I use the Bookmark Manager.

**Comment by sbernecchia** [comment]
any news?

**Comment by nicolaipre** [comment]
Following. This is a very annoying issue if you have a bit of bookmarks.

Any updates here, @brave ?

**Comment by lordparvis** [comment]
this issue is still here and we are waiting for the solution/bug fix

**Comment by andkonarcade** [comment]
Can confirm, very annoying.

**Comment by babzo007** [comment]
Is it possible that brave uses chromes engine or something like that? Because i have the same bug when using chrome :(

**Comment by andkonarcade** [comment]
Yes, both are based off Chromium.

**Comment by babzo007** [comment]
> Yes, both are based off Chromium.

Ok, then it may never be solved unless it is fixed with chromium I guess 😳

**Comment by the-overdriven** [comment]
Incredibly annoying. Any news on a possible fix or workaround?

**Comment by speedyhoon** [comment]
Any updates [@brave](https://github.com/brave)? The Bookmark Manager has been unusable for [4 years](https://community.brave.com/t/sync-v1-refreshes-brave-bookmarks-and-page-looses-scroll-state/132323)!

**Comment by CarlosMtnez** [comment]
Reorganizing the bookmarks is very annoying; the Bookmarks manager is almost unusable.

**Comment by lordparvis** [comment]
It's **July 7th**, and this issue is still here and we are **waiting** for the solution/bug fix

**Comment by Shutruk** [comment]
2024 and we still have this bug...

**Comment by andkonarcade** [comment]
> 2024 and we still have this bug...

Unfortunately, yes.

**Comment by lordparvis** [comment]
@andkonarcade @Shutruk It's fixed, been a month or more since then.

**Comment by Shutruk** [comment]
> @andkonarcade @Shutruk It's fixed, been a month or more since then. 

Just had the bug before that comment, that's how I found about it.

**Comment by lordparvis** [comment]
> > @andkonarcade @Shutruk It's fixed, been a month or more since then.
> 
> Just had the bug before that comment, that's how I found about it.

Which platform are you on ? Cause Brave stable, beta, nightly dont have this bug anymore.

**Comment by andkonarcade** [comment]
> > > @andkonarcade @Shutruk It's fixed, been a month or more since then.
> > 
> > 
> > Just had the bug before that comment, that's how I found about it.
> 
> Which platform are you on ? Cause Brave stable, beta, nightly dont have this bug anymore. Just tested Windows 10 and Mac versions, it works fine without jumping to top

I'm also using Windows 10 [Version 1.69.153 Chromium: 128.0.6613.85 (Official Build) (64-bit)]. The bookmark manager just jumped back to the top when I deleted a bookmark. Three times. Here is a quick video clip of the fourth time this happened just now:

https://github.com/user-attachments/assets/eaa27cba-f203-4476-9480-6aaced2b37d9

**Comment by lazymonkey2** [comment]
yes, still happening to me too (brave current version, win10)

**Comment by ErinsMatthew** [comment]
Same.  Still happens for me:

Brave Version 1.70.126 Chromium: 129.0.6668.100 (Official Build) (arm64)
macOS Sonoma Version 14.6.1 (23G93)

**Comment by binfordn** [comment]
I am also experiencing this issue. I was excited to try switching to Brave, but unfortunately this is a dealbreaker for me as I have hundreds of bookmarks that I need to keep organized.

Brave Version 1.73.91 Chromium: 131.0.6778.85 (Official Build) (64-bit)
Windows 11 Home 23H2 Build 22631.4460

**Comment by lordparvis** [comment]
turning off the sync chain fixed the issue for me

**Comment by fuesec** [comment]
still happning. latest macOS and latest Brave version. it's been happening for a while as in years

**Comment by lordparvis** [comment]
> still happning. latest macOS and latest Brave version. it's been happening for a while as in years

do you have synch chain on?

**Comment by Kanlep** [comment]
This is exactly what made me give up on Brave and go back to Firefox.
Unacceptable.

**Comment by PorthosSam** [comment]
I guess I am a bit more bogged down in inertia, gotten numb to it, or something. I haven't yet dropped Brave - hadn't thought about it for a while. Maybe now you've reminded me!    On Wednesday, July 30, 2025 at 07:11:40 PM MDT, Kanlep ***@***.***> wrote:  
 
 Kanlep left a comment (brave/brave-browser#16811)
This is exactly what made me give up on Brave and go back to Firefox.
Unacceptable.

**Comment by the-overdriven** [comment]
In Brave's defense: this isn't a Brave-specific bug, it also occurs in Chrome/Chromium.

I've found a workaround: right after editing or deleting a bookmark, quickly selecting any other bookmark leaves your current scroll position unchanged.

**Comment by andkondev** [comment]
> This is exactly what made me give up on Brave and go back to Firefox. Unacceptable.

I left Firefox for Brave years ago because it was a mess on desktop and mobile for me.



> In Brave's defense: this isn't a Brave-specific bug, it also occurs in Chrome/Chromium.
> 
> I've found a workaround: right after editing or deleting a bookmark, quickly selecting any other bookmark leaves your current scroll position unchanged.

This does not work all the time, at least it has not in the past.
